src/vm/hints/secp/secp_utils.rs

```diff
@@ -25,6 +26,24 @@ pub fn split(integer: &BigInt) -> Result<Vec<BigInt>, VirtualMachineError> {
     }
     Ok(canonical_repr)
 }
+
+/*
+Takes an UnreducedBigInt3 struct which represents a triple of limbs (d0, d1, d2) of field
+elements and reconstructs the corresponding 256-bit integer (see split()).
+Note that the limbs do not have to be in the range [0, BASE).
+prime should be the Cairo field, and it is used to handle negative values of the limbs.
+*/
+pub fn pack(d0: &BigInt, d1: &BigInt, d2: &BigInt, prime: &BigInt) -> BigInt {
+    let unreduced_big_int_3 = vec![d0, d1, d2];
+    let base: BigInt = bigint!(1) << 86_usize;
+
+    unreduced_big_int_3
+        .iter()
+        .enumerate()
+        .map(|(idx, value)| as_int(value, prime) * base.pow(idx as u32))
+        .sum()
```

Comment:
`(x^n)^m == x^(n*m)` so `(2⁸⁶)^n == 2^(86*n) == 2 << (86*n)`:
```suggestion
    let unreduced_big_int_3 = vec![d0, d1, d2];

    unreduced_big_int_3
        .iter()
        .enumerate()
        .map(|(idx, value)| as_int(value, prime) * (bigint!(1) << (idx * 86)))
        .sum()
```